ffi/src/proposal.rs

```diff
@@ -97,8 +98,15 @@ impl ProposalHandle<'_> {
 
         Ok(hash_key)
     }
-}
 
+    /// Creates an iterator on the proposal starting from the given key.
+    #[must_use]
+    #[allow(clippy::missing_panics_doc)]
+    pub fn iter_from(&self, first_key: Option<&[u8]>) -> CreateIteratorResult<'_> {
+        let it = self.iter_option(first_key).expect("infallible");
```

Comment:
The expect message should be more meaningful. I'd go so far as to refer to the new issue in the string "infallible; see issue NNNN"

another option is adding "// TODO: iter_option cannot fail; see issue NNNN"